misc/testerina/modules/testerina-runtime/build.gradle

```diff
@@ -53,13 +54,3 @@ dependencies {
 description = 'Ballerina - Test Executer'
 
 ext.moduleName = 'io.ballerina.testerina.runtime'
-
-compileJava {
-    inputs.property("moduleName", moduleName)
-    doFirst {
-        options.compilerArgs = [
-                '--module-path', classpath.asPath,
-        ]
-        classpath = files()
-    }
-}
```

Comment:
Lets add new line here

Reply:
Here actually I removed the trailing new line not found error by adding a new line at the end of this file.